exp/tests/test_response_postsave.py

```diff
@@ -208,3 +211,52 @@ def test_responses_per_study_type(self):
             .first()
             .study_type.is_ember_frame_player
         )
+
+
+class JspsychResponseSaveHandlingTestCase(TestCase):
+    def setUp(self):
+        jspsych = StudyType.get_jspsych()
+        user = G(User)
+        child = G(Child, birthday=datetime.date(1911, 11, 11), user=user)
+        study = G(Study, study_type=jspsych)
+        frame_id = "0-survey"
+        self.response = G(
+            Response,
+            child=child,
+            study=study,
+            study_type=jspsych,
+            sequence=[frame_id],
+        )
+        G(
+            Video,
+            response=self.response,
+            study=study,
+            s3_timestamp=timezone.now(),
+            frame_id=frame_id,
+        )
+
+    def test_exit_withdraw_videos(self):
+        settings.CELERY_TASK_ALWAYS_EAGER = True
```

Comment:
Does this need to be set back to `False` at the end of the test or in a `tearDown` function?

Reply:
It didn't change the outcome of the tests, but I did it anyway.

Reply:
Thanks!